### PR TITLE
Increase global create modal delay to 1500ms

### PR DESCRIPTION
**Pull request by roryabraham**
### Details
Hopefully temporary workaround until https://github.com/Expensify/App/issues/5296 is resolved.

### Fixed Issues

$ https://github.com/Expensify/App/issues/5303

### Tests / QA Steps

1. Log in with a user that's never signed into New Expensify before.
    - Dev-only tip: Emulate this by setting the following NVP:
    
        ```sql
        UPDATE nameValuePairs SET value = 'true' WHERE name = 'isFirstTimeNewExpensifyUser' AND accountID = INSERT_ACCOUNT_ID_HERE;
        ```
1. Ensure that the global create popover menu is correctly displayed.
1. Log back out and sign in with the same user.
1. The global create popover menu should not automatically open.

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [ ] Android

#### Web
https://user-images.githubusercontent.com/47436092/134079633-f6d6fcac-d63c-43bc-bbce-4d6fb2cd1232.mov

#### Mobile Web
https://user-images.githubusercontent.com/47436092/134079965-e11da4bf-fdee-4b56-a006-1141280daf5f.mov

#### Desktop
https://user-images.githubusercontent.com/47436092/134080260-0cdff4f3-f54f-4bd5-85cb-c2af947fc810.mov

#### iOS
https://user-images.githubusercontent.com/47436092/134077604-814ac4c0-1944-4e25-801f-c10d9a177561.mov

### CODE REVIEW

File: src/pages/home/sidebar/SidebarScreen.js [hunk 1]
```diff
@@ -57,15 +58,24 @@ class SidebarScreen extends Component {
...
-            // For some reason, the menu doesn't open without the timeout
-            setTimeout(() => {
-                this.toggleCreateMenu();
-
-                // Set the NVP back to false (this may need to be moved if this NVP is used for anything else later)
+        // NOTE: This setTimeout is required due to a bug in react-navigation where modals do not display properly in a drawerContent
+        // This is a short-term workaround, see this issue for updates on a long-term solution: https://github.com/Expensify/App/issues/5296
+        setTimeout(() => {
+            if (this.props.isFirstTimeNewExpensifyUser) {
+                const hasFreePolicy = _.chain(this.props.policies)
```
Comment (Jag96) [inline src/pages/home/sidebar/SidebarScreen.js:65]: We reverted this code in https://github.com/Expensify/App/pull/5360/files#diff-9b9fee594bc95324fa21caa02ac4d6d9692ce460c56587ff9aea35893fb3dde2L82, is this up to date?

Comment (roryabraham) [reply src/pages/home/sidebar/SidebarScreen.js:65]: Sorry, bad merge. Will fix and retest.

**Review by luacmartins — APPROVED** [review]
LGTM and tests well.

**Review by Jag96 — APPROVED** [review]

**Comment by roryabraham** [comment]
Should we CP this? When I created this the checklist was still unlocked.

**Comment by Jag96** [comment]
Ah good point, yeah I can CP manually

**Pull request merged**